webpack/ForemanInventoryUpload/Components/PageHeader/__tests__/__snapshots__/PageHeader.test.js.snap

```diff
@@ -4,13 +4,13 @@ exports[`PageHeader rendering render without Props 1`] = `
 <div
   className="inventory-upload-header"
 >
-  <ConnectedSettingsWarning />
-  <PageTitle />
+  <Component />
```

Comment:
Why do we still need the snaps? We should be able to remove them, if we use the RTL way